File: redux-realtime/src/components/Review.js
import React from 'react'
import PropTypes from 'prop-types'

const Review = ({ artist, album, rating, link }) => (
    <tr>
        <td>{artist}</td>
        <td>{album}</td>
        <td>{rating}</td>
        <td><a href={link}>Full review</a></td>
    </tr>
)

Review.propTypes = {
    artist: PropTypes.string.isRequired,
    album: PropTypes.string.isRequired,
    rating: PropTypes.number.isRequired,
    link: PropTypes.string.isRequired
}

export default Review;

File: sputnik/Review.py
class Review:
    def __init__(self, score, link, artist, album):
        self.score = score
        self.link = link.encode('utf-8')
        self.artist = artist.encode('utf-8')
        self.album = album.encode('utf-8')

    def __str__(self):
        return "{0} - {1}: Scored a {2} / 5.0. Review at {3}".format(self.artist, self.album, self.score, self.link)

    def serialize(self):
        return {
            'artist': self.artist,
            'album': self.album,
            'rating': self.score,
            'link': self.link
        }
